titiler/pgstac/db.py

```diff
@@ -1,16 +1,21 @@
 """Database connection handling."""
 
+from typing import Optional
+
 from psycopg_pool import ConnectionPool
 
 from titiler.pgstac.settings import PostgresSettings
 
 from fastapi import FastAPI
 
-settings = PostgresSettings()
-
 
-async def connect_to_db(app: FastAPI) -> None:
+async def connect_to_db(
+    app: FastAPI, settings: Optional[PostgresSettings] = None
```

Comment:
just added `optional` type information (which is how we generally do in this project 🤷)